modules/nvidia-gpu-admin-dashboard-using.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * monitoring/nvidia-gpu-admin-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-admin-dashboard-using_{context}"]
+= Using the NVIDIA GPU administration dashboard
+
+After deploying the the OpenShift Console NVIDIA GPU plugin, log in to the OpenShift Container Platform web console using your login credentials to access the *Administrator* perspective.
+
+== Viewing the cluster GPU overview
+
+You can view the status of your cluster GPUs in the Overview page by selecting
+Overview in the Home section.
+
+The Overview page provides information about the cluster GPUs, including:
+
+* Details about the GPU providers
+* Status of the GPUs
+* Cluster utilization of the GPUs
+
+== Viewing the GPUs dashboard
+
+You can view the NVIDIA GPU administration dashboard by selecting GPUs
+in the Compute section of the OpenShift Console.
+
+Charts on the GPUs dashboard include:
+
+* GPU utilization
+* Memory utilization
+* Encoder utilization
+* Decoder utilization
+* Power consumption
+* GPU temperature
```

Comment:
This shows the current GPU temperature and is based on [this](https://docs.nvidia.com/datacenter/dcgm/latest/dcgm-api/dcgm-api-field-ids.html#c.DCGM_FI_DEV_GPU_TEMP) metric.

The maximum is set to `110`, which is an empirical number, as the actual number is not exposed via a metric.